bitnami/drupal/10/debian-11/rootfs/opt/bitnami/scripts/libdrupal.sh

```diff
@@ -189,6 +194,55 @@ drupal_initialize() {
     true
 }
 
+########################
+# Installs dependencies with Composer
+########################
+drupal_composer_install() {
+    composer_file_dir="$(dirname "${DRUPAL_COMPOSER_FILE:-}")"
+    composer_lock_file="${composer_file_dir}/composer.lock"
```

Comment:
I think `DRUPAL_MOUNTED_COMPOSER_LOCK_FILE` should be used here, or am I wrong ?